howto/update/upgrade-anbox.md

```diff
@@ -37,39 +62,87 @@ The deployed Juju charms need to be upgraded next.
-    juju upgrade-charm etcd --revision=<rev>
-    juju upgrade-charm --channel=1.17/stable lxd
-    juju upgrade-charm --channel=1.17/stable ams
-    juju upgrade-charm --channel=1.17/stable ams-node-controller
-    juju upgrade-charm --channel=1.17/stable aar
+    juju refresh --channel=1.17/stable ams
 
-If you have the streaming stack deployed you have to upgrade also the following charms:
+### Upgrade LXD
 
-    juju upgrade-charm --channel=1.17/stable anbox-stream-gateway
-    juju upgrade-charm --channel=1.17/stable anbox-stream-agent
-    juju upgrade-charm --channel=1.17/stable coturn
-    juju upgrade-charm nats
+As last step you have to upgrade the LXD cluster. Upgrading LXD will cause no restart of running containers but it's recommended to take a backup before continueing.
 
-Once the commands are executed, Juju will perform all necessary upgrade steps automatically.
+As first step you need to upgrade the AMS node controller by running
 
-After Juju has settled the workload status will be marked as `blocked` and the status will show `UA token missing`.
+    juju refresh --channel=1.17/stable ams-node-controller
 
-Anbox Cloud requires a valid Ubuntu Pro token for an Ubuntu Pro subscription which includes the Anbox Cloud entitlement. You can get your Ubuntu Pro token on [Ubuntu Pro](https://ubuntu.com/pro). Please speak with your Canonical account representative.
+Once the upgrade completed, you can continue upgrading LXD:
 
-When you have your Ubuntu Pro token, you can apply it for all relevant charms with the following commands:
+    juju refresh --channel=1.17/stable lxd
 
-    juju config ams ua_token=<your token>
-    juju config lxd ua_token=<your token>
-    juju config ams-node-controller ua_token=<your token>
-    juju config aar ua_token=<your token>
-    juju config anbox-stream-gateway ua_token=<your token>
-    juju config anbox-stream-agent ua_token=<your token>
-    juju config anbox-cloud-dashboard ua_token=<your token>
+In case you want to upgrade individual LXD nodes separately, you can set the following configuration option for the LXD charm before running the refresh command above:
+
+    juju config lxd enable_manual_upgrade=true
+
+This will allow you to run the actual upgrade process for each deployed LXD instance separately. After the charm has been refreshed, the upgrade for a single LXD deployment unit can be triggered by running
+
+    juju run --wait=30m lxd/0 upgrade
+
+Once the upgrade has completed the unit will be marked as active.
+
+For major and minor version upgrades, an update of the LXD charm may upgrade kernel modules or GPU drivers. This requiers stopping any running containers before applying the upgrade and performing a reboot of the machine once the upgrade completed.
+
+In case a reboot of the machine is required a status message will be shown. When the machine has been rebooted, the status message can be cleared by running
+
+    juju run --wait=1m lxd/0 clear-notification
```

Comment:
```suggestion
Once the upgrade has completed, the unit will be marked as active.

For major and minor version upgrades, an update of the LXD charm may upgrade kernel modules or GPU drivers. This requires stopping any running containers before applying the upgrade and performing a reboot of the machine once the upgrade completed.

In case a reboot of the machine is required, a status message will be shown. When the machine has been rebooted, the status message can be cleared by running:

    juju run --wait=1m lxd/0 clear-notification
```